src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -584,4 +597,21 @@ private function getFileLink(string $absolutePath): string
 
         return (string) $this->fileLinkFormatter->format($absolutePath, 1);
     }
+
+    private function getAvailableNameArguments(string $name): array
+    {
+        $names = [];
+        $paths = $this->getLoaderPaths();
+
+        foreach ($paths as $namespace => $paths) {
+            $names[] = $namespace;
+        }
+
+        return $names;
```

Comment:
This can be replaced with `array_keys($this->getLoaderPaths())` (and can be inlined on line 118 directly)